packages/sdk/src/interactions/connector.ts

```diff
@@ -82,6 +82,7 @@ interface ConfigParameterTypes {
     onViewModeChanged: (viewMode: string) => void;
     onBarcodeValidationChanged: (validationResults: string) => void;
     onDataSourceIdChanged: (connectorId?: Id) => void;
+    onDocumentIssueListEvent: (documentIssues: string) => void;
```

Comment:
Nit, I don't find it wrong because it is a list of `DocumentIssue` type but `onDocumentIssuesListEvent` sounds more explicit to me.

Reply:
Pinging you, related to the naming discussion we had @evan-mcgeek @NicolaVerbeeck

Reply:
is this subscription any different in the first place? no `changed` in its name like in all others?